src/extensions/averagewaveforms/averagewaveforms.tsx

```diff
@@ -15,12 +15,14 @@ export function activate(context: ExtensionContext) {
     context.registerSortingView({
         name: 'AverageWaveformsNew',
         label: 'Average waveforms new',
-        component: AverageWaveformsNew
+        component: AverageWaveformsNew,
+        singleton: true
```

Comment:
We tag a sorting view plugin as singleton if it should only appear once in the MountainView (or similar) gui's